src/screens/ProfileScreen/elements/BioAndDMSetting.tsx

```diff
@@ -18,6 +18,46 @@ type BioAndDMSettingProps = {
   onlyReceivedDmFromUserFollowing: boolean;
 };
 
+const CheckBoxCustom = (props: {value: boolean; label: string}) => {
+  return (
+    <View
+      style={{
+        display: 'flex',
+        flexDirection: 'row',
+        alignItems: 'center'
+      }}>
+      <View
+        style={{
+          width: 16,
+          height: 16,
+          borderRadius: 9999,
+          backgroundColor: COLORS.white,
+          marginRight: 5
+        }}>
+        <CheckBox
+          value={props.value}
+          onCheckColor={COLORS.white}
+          tintColors={{true: COLORS.signed_primary, false: COLORS.white}}
+          tintColor={COLORS.signed_primary}
+          onTintColor={COLORS.signed_primary}
+          onFillColor={COLORS.signed_primary}
+          style={{
+            width: 16,
+            height: 16
+          }}
+        />
+      </View>
+      <Text
+        style={{
+          fontSize: 12,
+          fontWeight: '500'
+        }}>
+        {props.label}
+      </Text>
+    </View>
+  );
+};
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The custom CheckBox component `CheckBoxCustom` is well implemented. It's a good practice to create reusable components like this one. However, consider moving this component to a separate file in a shared components directory. This would improve modularity and allow other parts of the application to reuse this component if needed.



<!-- This is an auto-generated comment by OSS CodeRabbit -->